apps/mobile/src/screens/SettingsScreen/SettingsScreen.tsx

```diff
@@ -90,7 +88,7 @@ export function SettingsScreen() {
   }, [query.viewer?.user?.roles]);
 
   return (
-    <View style={{ paddingTop: top }} className="relative flex-1 bg-white dark:bg-black-900">
+    <View className="relative flex-1 bg-white dark:bg-black-900">
```

Comment:
it looks like we still need a little bit of top padding 

![CleanShot 2023-09-11 at 14 46 31@2x](https://github.com/gallery-so/gallery/assets/80802871/5897177a-2b8c-4591-9135-642d1804da51)

Reply:
Fixed!

<img width="446" alt="CleanShot 2023-09-11 at 15 57 10@2x" src="https://github.com/gallery-so/gallery/assets/4480258/24413fd7-1641-4d43-b1d4-ea804993add4">